packages/api/src/external/hie/validate-patient-links.ts

```diff
@@ -0,0 +1,46 @@
+import { normalizePatientInboundMpi } from "@metriport/core/mpi/normalize-patient";
+import { PatientData } from "@metriport/core/domain/patient";
+import { epicMatchingAlgorithm, strictMatchingAlgorithm } from "@metriport/core/mpi/match-patients";
+import { isStrictMatchingAlgorithmEnabledForCx } from "@metriport/core/external/aws/app-config";
+
+const SIMILARITY_THRESHOLD = 8.5;
+
+// TODO #2641 this is a temporary solution to validate the links belong to the patient
+// we need to create a more robust solution to validate the links belong to the patient
+export async function validateLinksBelongToPatient<NetworkLink>(
```

Comment:
This `NetworkLink` is not giving any type information for this function, so I'm assuming it's all being referred to as `any`?

If that's the case, it would be good to have a type that we can use here so the code inside this function is not using `any`, which is very prone to runtime errors.

Reply:
Split this into 2 separate functions